python/tvm/relay/frontend/keras.py

```diff
@@ -461,6 +479,8 @@ def _convert_concat(inexpr, keras_layer, _):
 
 def _convert_reshape(inexpr, keras_layer, _):
     _check_data_format(keras_layer)
+    if len(keras_layer.target_shape) < 3:
+        return _op.reshape(inexpr, newshape=(1, ) + keras_layer.target_shape)
```

Comment:
Can you add a test where this change is necessary?

Reply:
@kazum test added